### ISSUE TITLE
GKE support

### ISSUE THREAD

**Issue by rimusz** [ClusterHQ/flocker#2285]
Would be nice to have GKE support
I know it is Kubernetes cluster run by Google, but having Flocker working there would be very nice.

**Comment by wallnerryan** [comment]
Hi @rimusz we are currently working on support for GCE. Flocker works with Kubernetes today, so once the GCE bit is complete we can have flocker with with kubernetes and GCE. I will keep this issue updated as we work on it. You can also track GCE support here https://clusterhq.atlassian.net/browse/FLOC-1440

**Comment by rimusz** [comment]
@wallnerryan any updates on the matter?

**Comment by nVitius** [comment]
@rimusz  Someone mentioned in another issue that the progress for GCE support would be tracked here: https://clusterhq.atlassian.net/browse/FLOC-1149

**Comment by wallnerryan** [comment]
GCE support was announced today with the release of 1.11 https://clusterhq.com/2016/04/06/flocker-update-v1-11/

**Comment by sourcec0de** [comment]
@wallnerryan what are the remaining hurdles of running flocker on GKE now that GCE support has been completed?

**Comment by wallnerryan** [comment]
@sourcec0de I think one of the hurdles we're investigating is running flocker as a native deamonset on kubernetes. Sinec GKE brings up special instances, installing packages would need to be automated seperately, though having it natively work as a daemonset it one way. Nothing is slated for release and I can't give any dates, but we are interested.

**Comment by sourcec0de** [comment]
@wallnerryan just checking in on this again.

**Comment by adamtheturtle** [comment]
@sourcec0de Unfortunately ClusterHQ has shut down and therefore this repository is not being maintained. Some ex-ClusterHQ engineers have formed ScatterHQ. See http://scatterhq.com - we are interested in finding partners who will work with us to continue the development of Flocker.
